src/en/config-vmware.md

```diff
@@ -0,0 +1,51 @@
+
+Title: Configuring Juju for use with vSphere
+
+# Configuring the VMWare (vSphere) Provider
+
+Juju supports VMWare's vSphere ("Software-Defined Data Center") installations
+as a targetable cloud. In order to use the vSphere provider, you will need to
+have an existing vSphere installation, which supports VMWare's Hardware Version
+8 or better.
+
+
+To enable Juju to work with VMWare vSphere, you should start by generating a
+generic configuration file using the command:
+
+```bash
+juju generate-config
+```
+
+This will generate a file, `environments.yaml`, which will live in your
+`~/.juju/` directory (and will create the directory if it doesn't already
+exist).
+
+!!! Note: If you have an existing configuration, you can use
+`juju generate-config --show` to output the new config file, then copy and
+paste relevant areas in a text editor etc.
+
+The basic configuration will look something like this:
+
+```yaml
+ vsphere:
+    type: vsphere
+    host: <api-endpoint>
+    user: <some-user>
+    password: <some-password>
+    datacenter: <datacenter-name>
+    external-network: <external-network-name>
+```
+
+The values contained within angle brackets (`<` and `>`) need to be replaced
```

Comment:
yes. part of the problem is that this isn't standardised in the boilerplate files themselves, which is something I guess we will have to fix